deps/build.jl

```diff
@@ -47,7 +47,7 @@ if gitver == "notfound"
         isfile(filename) || run(download_cmd(downloadurl, filename))
         # TODO: checksum validation
         (b, ext, sec_ext) = splittarpath(filename)
-        run(unpack_cmd(filename, dest, ext, sec_ext))
+        run(unpack_cmd(filename, dest, is_windows() ? ".7z" : ext, sec_ext))
```

Comment:
This particular file happens to be a self-extracting `.7z.exe`, but BinDeps doesn't recognize that extension pair as being handled by 7z yet. And I didn't feel like figuring out the default extraction path or how to change it.